abdn/socialcomp/.htaccess

```diff
@@ -0,0 +1,51 @@
+Options +FollowSymLinks
+
+# Turn off MultiViews
+
+Options -MultiViews
+
+
+
+# Directive to ensure *.rdf files served as appropriate content type,
+
+# if not present in main apache config
+
+AddType application/rdf+xml .rdf
+
+AddType application/rdf+xml .owl
+
+AddType text/turtle .ttl
+
+
+
+RewriteEngine on
+
+
+
+#Rewrite rules for SC-PROV vocabulary extension 
+
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+
+RewriteRule ^sc-prov$ https://github.com/m-markovic/SC-PROV-Ontology/SC-PROV_Doc/  [R=303,NE,L]
+
+RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+
+RewriteRule ^asteroids$ https://raw.githubusercontent.com/m-markovic/SC-PROV-Ontology/master/sc-prov.owl [R=303,NE,L]
```

Comment:
Sorry, 

I was using https://github.com/perma-id/w3id.org/blob/master/games/spec/.htaccess <https://github.com/perma-id/w3id.org/blob/master/games/spec/.htaccess> as a template… 

whitespace lines must have gotten in because of copy and paste from Komodo Edit on Mac. Shoudl be ok now.

Milan 


> On 19 Feb 2016, at 16:10, David I. Lehn <notifications@github.com> wrote:
> 
> In abdn/socialcomp/.htaccess <https://github.com/perma-id/w3id.org/pull/275#discussion_r53480032>:
> 
> > +
> > +RewriteCond %{HTTP_ACCEPT} text/html [OR]
> > +
> > +RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
> > +
> > +RewriteCond %{HTTP_ACCEPT} text/\* [OR]
> > +
> > +RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
> > +
> > +RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
> > +
> > +RewriteRule ^sc-prov$ https://github.com/m-markovic/SC-PROV-Ontology/SC-PROV_Doc/  [R=303,NE,L]
> > +
> > +RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
> > +
> > +RewriteRule ^asteroids$ https://raw.githubusercontent.com/m-markovic/SC-PROV-Ontology/master/sc-prov.owl [R=303,NE,L]
> Copy & paste error?
> Also, why all the whitespace lines in the file?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/perma-id/w3id.org/pull/275/files#r53480032>.
>